luigi/contrib/mongodb.py

```diff
@@ -35,6 +35,9 @@ def __init__(self, mongo_client, index, collection):
         self._index = index
         self._collection = collection
 
+    def __str__(self):
+        return '%s/%s' % (self._index, self._collection)
```

Comment:
personally, i'd prefer the use of `str.format()` or f-strings

Reply:
```suggestion
        return f'{self._index}/{self._collection}'
```